packages/releaser/config/services.yaml

```diff
@@ -0,0 +1,25 @@
+parameters:
+    monorepo_package_name: "shopsys/shopsys"
+
+    travis_packages:
```

Comment:
It would be nice to provide some documentation for these parameters - I guess that this is used for configuring `CheckPackagesTravisBuildsReleaseWorker::$travisPackages` but I have no idea how does this work - there is some magic behind this? :slightly_smiling_face:

Reply:
the same question applies to `monorepo_package_name` and `lock_version_allowed_packages`

Reply:
Well, it's YAML convention to `use_underscore` and `$phpToCamelCase`.

It's a Symplify compiler pass, that translates and autowires parameters. I explain [the logic in this post](https://www.tomasvotruba.cz/blog/2018/11/05/do-you-autowire-services-in-symfony-you-can-autowire-parameters-too/#autowired-parameters-in-symfony).